inclusive-design/idrc
# Content changes and backstop update

## Description

* Updating Jess' page
* Added IUI and ODD project content
* updating reference images for visual regression tests
* updated backstop version

## Steps to test

* Open projects page and follow link to see ODD and Virtual healthcare project pages. Project pages should look fine.
* Open Jess' page and see the content is rendering properly. /about/team/jess-mitchell/ (note: this page does not seem to be indexed by the site, but may be linked from elsewhere).
* backstop visual regression tests should pass.
